src/transformers/models/seggpt/configuration_seggpt.py

```diff
@@ -78,7 +78,7 @@ class SegGptConfig(PretrainedConfig):
             Whether to use relative position encoding in the Attention
         merge_index (`int`, *optional*, defaults to 2):
             The index of the encoder layer to merge the embeddings.
-        encoder_output_indicies (`List[int]`, *optional*, defaults to `[5, 11, 17, 23]`):
+        output_indicies (`List[int]`, *optional*, defaults to `[5, 11, 17, 23]`):
```

Comment:
```suggestion
        out_indices (`List[int]`, *optional*, defaults to `[5, 11, 17, 23]`):
```
Should be out_indices

Reply:
Fixed here and renamed in the other places as well https://github.com/huggingface/transformers/pull/27735/commits/5190205197d2f0790d5aeb84b960cc2eaef040bd